### ISSUE TITLE
why .container haven't padding

### ISSUE THREAD

**Issue by blackinitial** [jgthms/bulma#2249]
why `.container` haven't padding on mobile. i must wrap with `.section` so layout on mobile not over to right.

**Comment by blackinitial** [comment]
Thanks for reply, i'm just confused cz i'm used to using bootstrap.

**Comment by ogai** [comment]
Just in case anyone came here having the same confusion as me: On 0.8.2 either the tag _section_ nor the class _container_ has padding or margin on mobile. You must use the tag _section_ with the class _section_. See https://bulma.io/documentation/layout/section/ 

However, if you add the class _is-fluid_ together with the class _container_, you have a gap on all devices without the need for _section_.

I think this behavior is confusing.

**Comment by mwiest** [comment]
> Because the container is only here to keep the content centered passed a certain point. The section provides the padding.

I also think this is confusing. In my mental model, the container constraints the width of the content to a sensible maximum per screen size. On mobile, this should be with some padding on the side, which is the natural thing to have for actual content. Maybe not for bg-images or color, but that's something you'd apply outside the container anyway for it to work on larger screens, right? Can someone explain the reasoning behind this? I think .section is doing .container's job here.

**Comment by RichardJECooke** [comment]
> However, if you add the class is-fluid together with the class container, you have a gap on all devices without the need for section.

No, this doesn't work. If you use `<div class="container is-max-desktop is-fluid">` then the container is out of alignment. If you use `<div class="container is-max-desktop mx-4">`  the container is out of alignment. If you use `<div class="section"><div class="container is-max-desktop ">`  it works, but also adds a top margin. This is the best solution, but still not ideal.

There seems to be no way to add just a left and right margin to a container and still have it not too wide on desktop screens.

**Comment by RichardJECooke** [comment]
Also, this really should be mentioned in the docs. As a new Bulma user I'm finding so many undocumented unnecessary changes to normal CSS, like the automatic adding of scrollbars to pages, changing heading sizes.
